pkg/controller/bootstrap/bootstrap.go

```diff
@@ -112,6 +116,10 @@ func (b *Bootstrap) Run(destDir string) error {
 				icspRules = append(icspRules, obj)
 			case *apicfgv1.Image:
 				imgCfg = obj
+			case *apicfgv1.FeatureGate:
+				if obj.GetName() == ctrlcommon.ClusterFeatureInstanceName {
```

Comment:
If I'm interpreting this correctly, since you are not storing this as an array, the expected behaviour is that the user only applies 1 config. On the off chance they supply multiple, it will not error but instead have the higher alphanumeric take priority?

Reply:
Yep, so the expectation is that a customer should only create one, whichever is read last will win, not sure if you want to have logic in here to account for that or not? But I'd expect a customer to only supply one.

Also note, it has to have a particular name, `cluster`, so that's why we expect a singleton here